src/components/EntityInfo/EntityMenu.vue

```diff
@@ -122,6 +98,28 @@
 
         <v-list dense>
           <v-list-item-group color="primary">
+            <!-- View/Add Digital Credentials -->
```

Comment:
There's a height difference with this item.

![image](https://github.com/bcgov/business-filings-ui/assets/10002889/b9ec8589-0e6c-4a77-be2f-54cf0f76a6ba)

Reply:
Should be fixed now

Reply:
Yep. Resolving.